relay-general/src/protocol/transaction.rs

```diff
@@ -123,6 +145,12 @@ pub struct TransactionInfo {
     /// This value will only be set if the transaction name was modified during event processing.
     #[metastructure(max_chars = "culprit", trim_whitespace = "true")]
     pub original: Annotated<String>,
+
+    /// A list of changes prior to the final transaction name.
+    pub changes: Annotated<Vec<Annotated<TransactionNameChange>>>,
```

Comment:
We should make extra clear to SDK devs that this list is only for subsequent changes of the transaction name. That is, it should _not_ contain an element for the initial value of the transaction name.